### DISCUSSION TITLE
netCDF in Plot Data Plane

### DISCUSSION THREAD

**Discussion by cjmelick** [dtcenter/METplus#1960] [Configuration] [answered]
Hi John,

I have a different question dealing with being unable to use plot_data_plane and trying to use a gridded field stored in a netCDF file.   If the error says that ".....nc not a valid data file", does that mean the header wasn't set up properly into being CF compliant always?.  Or are there exceptions to that rule?

Thanks,
Chris

**Comment by JohnHalleyGotway** [comment, marked as answer]
Hi @cjmelick, I see you have a question about MET not being able to read a gridded NetCDF file. The plot_data_plane tool prints that message on this [line of code](https://github.com/dtcenter/MET/blob/2bc783e4c6690a45f927b64b66e6f1ad927f8ba8/src/tools/other/plot_data_plane/plot_data_plane.cc#L141) when it's not able to handle the input NetCDF file. It's possible that the data file is corrupted and the NetCDF library is returning bad status. Or perhaps it's something else? If the filename uses an odd suffix, that may cause problems.

I'll need more information to provide a more useful answer.

If you point me to a sample data file, I could take a closer look and provide a better explanation. Here's [How to Send Us Data](https://github.com/dtcenter/METplus/discussions/954). Or just send me a link to it to download from elsewhere. Or point me to it on an NCAR/RAL machine or Cheyenne.

Or you could just send me the output of `ncdump -h` and I could make a guess without actually running anything.

**Reply by JohnHalleyGotway** [reply]
Just wanted to follow up on this discussion to see if questions remain. If not, please feel free to select the "Mark As Answer" button for one of the previous responses, and then I'll lock it to prevent future posts. If I don't hear otherwise in the next week or so, I'll just go ahead and mark it as being answered.

Thanks!

**Reply by cjmelick** [reply]
Hi John, sorry I didn't get back to you.   I am fine with closing it.   We are still investigating internally.    Thanks for your assistance!
